docs/testing/testplan.md

```diff
@@ -0,0 +1,427 @@
+   a. Any operating system (OS)
+   b. A chromium-based Web browser (e.g. Chrome, Edge, Brave) or Firefox
+
+### Reporting Issues
+
+1. Use any screen recording application to record the steps and results of your testing (e.g. Loom, Camtasia etc.).
+2. Log bugs or issues, and monitor their status and resolution on GitHub [link](https://github.com/interledger/web-monetization-extension/issues).
+3. Join the `#webmonetization` channel on [Slack](https://communityinviter.com/apps/interledger/interledger-working-groups-slack).
+
+### Prerequisites
+
+Before you start testing, sign-up for a wallet:
+
+1. The sign-up process depends on your digital wallet provider. Learn more from the [Dependencies](#Dependencies) section.
+2. If you want to use a test wallet, the sign-up process is quick (within minutes).
+3. If you want to use a real wallet:  
+   a. Your wallet provider will likely need to verify your identity (info: this is mandated by the laws of each country, or jurisdiction).  
+   b. The identity verification process depends on your wallet provider, and your location (info: this can be within hours or days).  
+   c. Once your wallet provider completes all sign-up and verification processes, you should be able to:
+   - Setup and manage your wallet accounts, currencies, balances etc.
+   - Obtain the wallet address or payment pointer for your digital wallet.
+
+### Test Playground
+
+Use the [Web Monetization playground](https://webmonetization.org/play/) to quickly create the conditions of a monetized website in a playground area, to use only for testing purposes.
+
+When you add a wallet address or payment pointer to the playground, it is added as a receiving wallet on the playground.  
+So, the playground behaves like a monetized website on which you can use the Web Monetization extension to pay.
+
+On the playground, you can add a wallet that uses real money from a real wallet, or you can add a wallet that uses “play” money from a “test wallet”.
+
+**Important**
+
+1. The playground itself is not web monetized. We do not receive any payments from you if you use this site.
+2. The playground becomes monetized when you add one or more receiving wallet addresses to it.
+3. If your Web Monetization extension is connected to a real wallet, then your extension will facilitate payments from your real wallet (i.e. real money).
+4. Similarly, if your extension is connected to a play wallet (e.g. rafiki.money), then the extension facilitates payments using play money.
+5. If you add your own wallet address to the playground, and then use the extension to pay the playground, you are paying yourself.
+
+### Test Data
+
+This section lists websites and wallets that you can pay using the Web Monetization extension.
+
+#### Use a test wallet & play money
+
+If you have a [rafiki.money](https://rafiki.money) test wallet address connected to your extension, then you can use the Web Monetization extension to pay with "play" money. You have two options:
```

Comment:
I see some links are reference often in this document. We can use markdown footer links here

```suggestion
If you have a [test wallet][testWallet] address connected to the extension, then you can use the Web Monetization extension to pay with "play" money. You have two options:
```

and at the bottom of the file:

```md
[testWallet]: https://rafiki.money
```
Or, with a title:

```md
[testWallet]: https://rafiki.money "rafiki.money"
```

Then when test wallet address changes, we've to update only at once place.